src/app/components/Modals/UnbondModal.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  DialogHeader,
+  Text,
+} from "@babylonlabs-io/bbn-core-ui";
+
+interface UnbondModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onProceed: () => void;
+}
+
+export const UnbondModal = ({
+  isOpen,
```

Comment:
Same here related to `isOpen` and `open` in #373

Reply:
will use isOpen from now on as its more descriptive